doc/conf.py

```diff
@@ -188,6 +188,7 @@
     ('py:attr', 'srcline'),
     ('py:class', '_StrPath'),  # sphinx.environment.BuildEnvironment.doc2path
     ('py:class', 'Element'),  # sphinx.domains.Domain
+    ('py:class', 'TextElement'),
```

Comment:
Some of the lines around this have a comment. Is that just the name of the first consumer? I can add `sphinx.application.Sphinx.connect` if preferred.